drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -18,7 +18,10 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
 
   delegate :exit_date, to: :exit, allow_nil: true
 
+  # CAUTION: enrollment.project accessor is overridden below
   belongs_to :project, **hmis_relation(:ProjectID, 'Project'), optional: true
+  has_one :client_project
```

Comment:
So can/should we get rid of the override function, and instead just call the `:client_project` relation `:project`? I thought that was what we discussed, since in the HMIS you _always_ wan't the Projects for WIP enrollments to be included.